guides/preprocessing_layers.py

```diff
@@ -273,19 +274,15 @@
 """
 
 # Define some toy data
-data = tf.constant(["a", "b", "c", "b", "c", "a"])
-
-# Use StringLookup to build an index of the feature values
-indexer = preprocessing.StringLookup()
-indexer.adapt(data)
+data = tf.constant([["a"], ["b"], ["c"], ["b"], ["c"], ["a"]])
```

Comment:
Can you remind me why this change is necessary? What was the previous behavior for `data = tf.constant(["a", "b", "c", "b", "c", "a"])` and what is the new behavior?

Reply:
It is not actually needed for the adapt data actually, but it is needed for the call time data to one-hot encode output (this is our inconsistent rank 1 behavior between CategoryEncoding and StringLookup issue).

I figured it would be simpler to keep all the inputs rank 2 for now. This goes away when we add "one_hot", we can go back to simple rank one inputs.